src/Map/src/Twig/MapRuntime.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Twig;
+
+use Symfony\UX\Map\Map;
+use Symfony\UX\Map\MapFactory;
+use Symfony\UX\Map\Renderer\RendererInterface;
+use Twig\Extension\RuntimeExtensionInterface;
+
+/**
+ * @author Simon André <smn.andre@gmail.com>
+ *
+ * @internal
+ */
+final class MapRuntime implements RuntimeExtensionInterface
+{
+    public function __construct(
+        private readonly RendererInterface $renderer,
+    ) {
+    }
+
+    /**
+     * @param Map|array<string, mixed> $map
+     * @param array<string, mixed>     $attributes
+     */
+    public function renderMap(Map|array $map, array $attributes = []): string
+    {
+        if (\is_array($map)) {
+            if (isset($map['attr'])) {
```

Comment:
I don't see where `attr` is defined, am I missing something? Why do we need this key if we already have `$attributes`?

Reply:
I'd like to offer a very simple function, like form ones ... as chaining to object in twig function calls is very midly readable..

So instead of 

```twig
{{  ux_map({ 
    center: ...
    zoom: ...
    markers: {
       ...
    }
}, {
   class: 'foo',
}) }} 
```

This allows

```twig
{{  ux_map({ 
    center: ...
    zoom: ...
    markers: {
       ...
    },
    attr: {
       class: 'foo',    
    }
}) }} 
```

The little things :)

Reply:
Thanks for the explanation, I can see a bit more clearly now.

But I'm not 100% fan, what about an alternative where we can abuse Twig named parameter? :
```twig
{{ ux_map(
    center={lat: 51.5074, lng: 0.1278},
    zoom=10,
    markers= [
        {lat: 51.5074, lng: 0.1278, title: 'London'},
        {lat: 48.8566, lng: 2.3522, title: 'Paris'},
        {lat: 40.7128, lng: -74.0060, title: 'New York'},
    ],
    attributes={
        class: 'foo',
        style: 'height: 400px; width: 100%;',
    }
) }}

```

Reply:
If this is the only thing, no problem, just remove the first `{ `and last `}`

```diff
- {{ ux_map({
+ {{ ux_map(
    center: ... 
    zoom: ... 
   markers: { ... }, 
    attr: { class: 'foo', } 
- }) }}
+ ) }}
```


(named parameters now uses "`:`" and this will be the recommanded code style)

Reply:
Well... not that simple... we need to keep "argumentes" as second arg of the function.